.github/workflows/end2end.yaml

```diff
@@ -408,6 +408,19 @@ jobs:
         env:
           STAGE: end2end-http
         if: always()
+      - name: "Debug: SSH to runner"
```

Comment:
There is already a `./.github/actions/debug-wait` action, which is currentyl useless (does not do anything).

So you should instead update that action, so that it becomes useful and so you factorize the ssh runner step(s) you are introducing (makes maintenance easier & simplifies main workflow)